Automattic/wp-calypso
# Stats Subscribers: Add Period Navigation

Related to https://github.com/Automattic/wp-calypso/issues/75002

## Proposed Changes

* This PR adds period navigation to the subscriber's chart. It is not currently fully functional, as the chart will need some additional changes (to use real data) in order to update visually based on the URL query string. However the period navigation should correctly update the URL query string. 

## Testing Instructions

* Open the Calypso Live Branch. 
* Navigate to `/stats/subscribers/{siteURL}?flags=stats/subscribers-section`
* Check the period navigation buttons (day/week/month/year etc) 

![image](https://user-images.githubusercontent.com/30754158/231660006-67341091-24aa-49aa-a609-a5780bf7458a.png)


## Pre-merge Checklist
